### PR TITLE
Do not transfer the MPI communicator when copying the triangulation

### PR DESCRIPTION
**Pull request by agrayver**
Follow-up of the discussion in #8969 

@bangerth @masterleinad FYI

### CODE REVIEW

**Comment by masterleinad** [comment]
```
invalid author 'agrayver' without firstname and lastname
```
Would you please fix your name for the commits?

File: include/deal.II/distributed/tria_base.h [hunk 1]
```diff
@@ -98,6 +98,11 @@ namespace parallel
 
     /**
      * Implementation of the same function as in the base class.
+     *
+     * @note this functions copies the cells, but not the communicator,
```
Comment (bangerth) [inline include/deal.II/distributed/tria_base.h]:
```suggestion
     * @note This functions copies the cells, but not the communicator, 
```

Comment (Rombur) [reply include/deal.II/distributed/tria_base.h]: There is also an extra `s` add the end of `function`

**Comment by agrayver** [comment]
Ready from my side.

**Comment by masterleinad** [comment]
CI is still failing:
```
invalid author 'agrayver' without firstname and lastname
```
We require to have proper first and last name in the commits.

**Comment by agrayver** [comment]
Well I am not sure how to fix this? Seemingly, it is all right:

> (base) ag@ag:~$ cat .gitconfig 
> [user]
> 	name = Alexander Grayver

**Comment by peterrum** [comment]
@agrayver  The problem is that your first commit still has `agrayver` as author. If you `rebase -i c223f4e0c4256f9f9ca9816500bf07fb25989bb2`, squash both commits and force push the changes, everything should work!

**Comment by masterleinad** [comment]
```
git commit --amend --author="John Doe <john@doe.org>"
```
should work in any way.

**Comment by peterrum** [comment]
> should work in any way.

That will only operate on the last commit, will it not? The problem is with the commit before...

**Comment by masterleinad** [comment]
That depends on the commit you are currently looking at. If you do an interactive rebase to fix a certain commit and mark it as `edit`, this is a way to change the author's information. I am not sure if calling `amend` without the flag is sufficient as well.

**Comment by agrayver** [comment]
I did everything as suggested above, but this still fails. Any clue?

**Comment by masterleinad** [comment]
I can take care of it.

**Comment by masterleinad** [comment]
Here you go.

**Comment by agrayver** [comment]
Thank you. Can we then merge it?

**Pull request merged**